turbinia/turbiniactl.py

```diff
@@ -636,7 +642,7 @@ def process_args(args):
           args=args, disk_name=disk_name, name=name, source=source,
           project=project, zone=zone, embedded_path=embedded_path,
           mount_partition=mount_partition, group_id=group_id,
-          filter_patterns=filter_patterns, client=client, yara_rules=yara_rules)
+          filter_patterns=filter_patterns, client=client, yara_rules=yara_rules, group_name=args.group_name, reason=args.reason, all_args=sys.argv)
```

Comment:
FYI, it looks like some of these lines are too long so it's failing the `yapf` tests.  See the test output for more details, but you can run something like `yapf --style .style.yapf -i -r /your/src/path/turbinia` and it should automatically fix some of these.